src/index.js

```diff
@@ -10,6 +10,7 @@ export * from './three/math/EllipsoidRegion.js';
 export * as GeoUtils from './three/math/GeoUtils.js';
 export * from './three/math/GeoConstants.js';
 export * from './three/math/OBB.js';
+export * from './three/math/TileBoundingVolume.js';
```

Comment:
What's the motivation for exporting this class? I think we can just export the types and not the class itself. The "Tile" object for each node is still fairly wild west - at some point it's going to make sense to go back and make that a bit more well-defined now that the project has solidified more.

Reply:
You are right